### ISSUE TITLE
Announcement: Brand/Logo Icon will be removed in v8

### ISSUE THREAD

**Issue by goyney** [Templarian/MaterialDesign#6602]
# **All brand and logo icons in the Material Design Icons library have been deprecated and will be _removed_ in version 8.**

This is a big moment for Material Design Icons and should come as no surprise based on the stance we've taken around brand icons since v5. Brand icons have always been tricky for us for a multitude of reasons, but the biggest issue we consistently face is licensing. When we deliver somebody else's copyrighted material in our library, it makes interpreting the license of the library as a whole very murky. Therefore, we've decided to remove that murkiness and reaffirm our goal: to provide a library that delivers [system icons created following Material guidelines](https://material.io/design/iconography/system-icons.html#design-principles).

## What icons will be removed?

All icons listed under our [Brand / Logo tag](https://materialdesignicons.com/tag/brand) have been deprecated and will be removed.

## When will these icons be removed?

This is a big change, so we want to give everyone ample time to act upon it. We've slated these icons to be removed in v8. As of the writing of this post, we are on v6.7. We average a major version PER YEAR, so it's likely these icons will stick around until sometime in mid-to-late-2023. As we get closer to v8, we'll make updates to the announcement with more solid dates.

## Can I still request new brand icons be added until v8?

No. We are no longer accepting any new brand or logo icons in Material Design Icons for any reason.

## What are my options to continue to use brand icons that are being removed?

1. You could stay on a version before v8. You won't get new icon additions however.
2. You can use a library like [Simple Icons](https://simpleicons.org/) -- a library built just for brand and logo icons. Our existing tooling works pretty well with their library out of the box.
3. You could check out our new brand icon library...

### Introducing the Pictogrammers Brand Icon Library

Obviously, there is a real need for a library of brand icons, otherwise we wouldn't have the ones we do or the constant requests for new ones. Therefore, the [Pictogrammers Icon Group](https://pictogrammers.com/), the group responsible for bringing you Material Design Icons, has begun to create our own [brand icon library](https://github.com/Pictogrammers/brand-icons)!

While it's not quite ready for use yet, by the time we remove the icons from MDI, it will be!

#### Why create another library?

One of the biggest struggles with including brand icons in any library is licensing. Other icon libraries out there who offer brand icons mostly infer permission based on wording in a brand's license or follow the "ask for forgiveness later" mantra. We want to do it differently.

Every brand icon we include in our new library will have been given permission by the brand owner, or the brand's license will explicitly state that their asset can be redistributed. We will provide all license, copyright, trademark, and branding guidelines (where available) for each and every glyph. When you use our new brand library, there will be no question about the license. If we cannot get permission and license information, we won't include the brand. It's that simple.

---

Any questions or concerns can be asked on this issue.

**Comment by icedterminal** [comment]
Could a link to this issue be added in the warning? For user experience purposes. Right now it's just a red text message with vague information. A link to this issue would greatly improve user experience and point users who need logos to the correct, upcoming source.

![image](https://user-images.githubusercontent.com/4755734/221543681-fbff6e83-8a67-4cfb-8c05-9da7dd60ecfc.png)

Thanks.

**Comment by mririgoyen** [comment]
@icedterminal We can deprecate icons for any number of reasons. Right now, it is a simple true/false flag. There might be room to improve that in the future. Could you [open a feature request on the website repo](https://github.com/Pictogrammers/pictogrammers.com/issues) to capture it?

**Comment by Secret-chest** [comment]
Please make another library with all brand icons, like YouTube or GitHub, and remove them if the owner asks you to.

**Comment by bobanum** [comment]
I don't know if that has already beeen asked and answered, but what will happen to icons that is a logo __AND__ a file type?
I saw "C++", "C", "C Sharp" and more non-brand icons deprecated. Will they be repurposed or just plain removed? How about "pdf" or "word" or eventually "photoshop/psd"? I would suggest leaving icons that also represent a file type or extension.

**Comment by fred-mevia** [comment]
Hello are you planning soon to include Alpha X logo (new twitter) et TikTok logo for social media in version 8 or before ? Thanks for all your amazing work. All the best!

**Comment by Secret-chest** [comment]
Well, the X is not copyrighted, it's just a mathematical X.

**Comment by RagingRoosevelt** [comment]
I noticed that the MIDI logo was being removed.  That make sense, given that it is a logo from a specific organization (MIDI Manufacturers Association).  

BUT!  shouldn't that also apply to stuff like the USB logo and the Bluetooth logo?  Both of those are standards that are licensed from a legal entity / standards body.  It seems like if it applies in some cases, it should be universal?

**Comment by Simran-B** [comment]
Good point. Maybe there is an argument to make for keeping icons that represent communication protocols as opposed to specific communication services and platforms.

**Comment by Secret-chest** [comment]
Then why not keep file type icons?

**Comment by Nuri0** [comment]
How is the state of this topic and the state of Material Design Icons as a whole?

The brand logos were planned to be removed with v8, which was expected for "mid-to-late-2023". We now have almost mid-2025 and are still on v7. I also noticed that the amount of changes (according to https://pictogrammers.com/library/mdi/history/) has decreased by a lot since the release of v7.4.47 in December 2023.

Is there still a release of v8 in planning which just takes longer than expected?

**Comment by darkgrayknight** [comment]
> How is the state of this topic and the state of Material Design Icons as a whole?
> 
> The brand logos were planned to be removed with v8, which was expected for "mid-to-late-2023". We now have almost mid-2025 and are still on v7. I also noticed that the amount of changes (according to https://pictogrammers.com/library/mdi/history/) has decreased by a lot since the release of v7.4.47 in December 2023.
> 
> Is there still a release of v8 in planning which just takes longer than expected?

Same kind of question: Are we adding any of the icons in the ["pipeline"](https://github.com/Templarian/MaterialDesign/milestone/35) that are ready to go? How do we get these added anytime soon?
